Sources/Date+Sugar.swift

```diff
@@ -12,9 +12,40 @@ extension Date {
         case date = "yyyy-MM-dd"
         case ISO8601 = "yyyy-MM-dd'T'HH:mm:ssZ"
     }
+
+    // MARK: Weekdays
+    enum Weekday: Int {
+        case sunday = 1
+        case monday
+        case tuesday
+        case wednesday
+        case thursday
+        case friday
+        case saturday
+    }
     
     // MARK: Manipulators
-    
+
+    /// Next
+    /// Take you to next weekday needed from the date given
```

Comment:
I think I'd change the wording of this comment to something like `returns the next  "weekday" starting from the provided date.`